.gitignore

```diff
@@ -6,3 +6,9 @@ Makefile.in
 autom4te.cache
 patches/*
 
+# Tool specific files
+.README.md.html
+.*.*~
+.project
+.settings
+.pydevproject
```

Comment:
Lines 9-10 add simple ignores for Eclipse and PyDev Eclipse Plugin based files, as well as GEdit files.